account_banking_payment_export/wizard/payment_order_create.py

```diff
@@ -45,9 +46,33 @@ def default_get(self, field_list):
     def extend_payment_order_domain(self, payment_order, domain):
         if payment_order.payment_order_type == 'payment':
             domain += [('account_id.type', 'in', ('payable', 'receivable')),
-                       ('amount_to_pay', '>', 0)]
+                       ('credit', '>', 0)]
```

Comment:
For several reasons:
# amount_residual_currency is not searchable
# it is done this way in the official module
# it would not be better than credit > 0 for the purpose of searching lines to pay